resources/output/route_table/aws_route_table.go

```diff
@@ -8,8 +8,8 @@ const AwsResourceName = "aws_route_table"
 
 type AwsRouteTable struct {
 	*common.AwsResource `hcl:",squash" default:"name=aws_route_table"`
-	VpcId               string               `hcl:"vpc_id,expr"`
-	Routes              []AwsRouteTableRoute `hcl:"route,blocks"`
+	VpcId               string               `hcl:"vpc_id,expr" json:"vpc_id,omitempty"`
```

Comment:
Perfect!

Can you do this for azure_route_table as well?

Reply:
done